ckanext/collaborators/tests/test_action.py

```diff
@@ -108,23 +108,6 @@ def test_delete_user_not_found(self):
             'dataset_collaborator_delete',
             id=dataset['id'], user_id=user['id'])
 
-    def test_deleting_user_removes_collaborator(self):
```

Comment:
@mariorodeghiero can we add this test again to the PR? This one probably got merged to master after you created the branch, so merging it will bring it back.